docs/source/getting-started/installation.rst

```diff
@@ -30,7 +30,11 @@ To install the SDK on Mac or Linux:
 Windows
 =======
 
-To install the SDK on Windows, download and run the installer from `github.com/digital-asset/daml/releases/latest <https://github.com/digital-asset/daml/releases/latest>`__.
+.. note::
+
+  There's currently a known issue where the installer gets flagged as "unrecognized", but this will be fixed soon.
```

Comment:
```suggestion
  There's currently a known issue where the installer gets flagged as "unrecognized" by Windows Defender, but this will be fixed soon.
```

That makes it clear which message is the one they can expect